Automattic/wp-calypso
# privacy: migrate `requestDpa` away from `http-data`

#### Changes proposed in this Pull Request

* Migrate `requestDpa` to use ReactQuery
* Refactor `DPA` section to be its own component which encapsulates any relevant code
* Disable `Request DPA` button in case a request is running

#### Testing instructions

* Go to `/me/privacy`
* Hit the `Request DPA` button, make sure you see a success notice
* Block network requests in your browser and make sure you see a general error notice
* Now make sure you're **not** proxied and hit the button more than 5 times, make sure you see an error notice about being throttled
